pkg/runner/run_context.go

```diff
@@ -225,7 +225,13 @@ func (rc *RunContext) newStepExecutor(step *model.Step) common.Executor {
 			common.Logger(ctx).Infof("  \u2705  Success - %s", sc.Step)
 		} else {
 			common.Logger(ctx).Errorf("  \u274C  Failure - %s", sc.Step)
-			rc.StepResults[rc.CurrentStep].Success = false
+
+			if sc.Step.ContinueOnError {
+				log.Infof("Failed but conitnue next step")
```

Comment:
Please use the same logging pattern as above. Also, check typo on 'continue'

Reply:
Thank you your comment.
Let me check my code, and I will revise it.

Reply:
I fixed code, Could you review this point again?